lib/Request.js

```diff
@@ -166,4 +165,22 @@ function extend(object, donor) {
     return object;
 }
 
+function isErrorStatusCode(response_error_code, error_status_codes) {
+
+    error_status_codes = error_status_codes.split(',');
+    for (var index = 0, len = error_status_codes.length; index < len; ++index) {
+        var error_code = error_status_codes[index];
+        if (error_code.indexOf('-') > -1) {
+            var from_to = error_code.split('-').map(function (ec) { return parseInt(ec)});
+            if (response_error_code >= from_to[0] && response_error_code <= from_to[1]) {
+                return true;
+            }
+        }
+        if (response_error_code === parseInt(error_code)) {
```

Comment:
Add [base 10](https://stackoverflow.com/questions/6611824/why-do-we-need-to-use-radix)